prometheus_juju_backup_all_exporter/collector.py

```diff
@@ -79,13 +92,15 @@ def _update_metrics(self):
         new_metrics = []
         for metric in self._metrics:
             new_metric = metric.__class__(
-                metric.name, metric.documentation, labels=metric._labelnames
+                metric.name,
+                metric.documentation,
+                labels=metric._labelnames,  # pylint: disable=W0212
```

Comment:
Using "private" attributes is discouraged for a good reason, Is there no other way to get these values? (Perhaps some property?)

Reply:
See above comment